classes/entity/enemy/koopa/parakoopa.gd

```diff
@@ -21,18 +21,24 @@ const color_presets = [
 
 @export var disabled = false: set = set_disabled
 @export var mirror = false
-@export var color: Koopa.ShellColor = Koopa.ShellColor.GREEN: set = set_color
+@export var color := Koopa.ShellColor.GREEN
 
 
 func set_color(new_color):
 	for i in range(3):
 		material.set_shader_parameter("color" + str(i), color_presets[new_color][i])
-	color = new_color
 	koopa.color = color
 	shell.color = color
 
 
 func _ready():
+	# Ensure that the material is unique so we can set its parameters.
+	# "Local to scene" causes issues with source control, because UIDs are refreshed on loading the scene.
+	# This method refreshes them at runtime instead of in the editor.
```

Comment:
I notice that you split material configuration off into `_init()` in `toad.gd`, but not here. Nothing against, I'm just curious.